repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/filter/AbstractConfiguredFilterTest.scala

```diff
@@ -143,6 +167,18 @@ class AbstractConfiguredFilterTest
 
       filter.getInitialized shouldBe true
     }
+
+    it("should call doConfigurationUpdated") {
```

Comment:
These are pretty basic tests. I'm more concerned with that they are calling in the right spot in the lifecycle versus them being called in general. Does that make sense?